e2e/api/restHelper.js

```diff
@@ -1,23 +1,11 @@
-const config = require('../config.js');
-
 const fetch = require('node-fetch');
-const HttpProxyAgent = require('http-proxy-agent');
-const HttpsProxyAgent = require('https-proxy-agent');
```

Comment:
can we remove these from package.json?

Reply:
ah yes, I guess they are no longer used now